modules/ROOT/pages/core-operations.adoc

```diff
@@ -210,19 +210,16 @@ The same is the case for `decr()`.
 
 A document may be used as a counter if its value is a simple ASCII number, like `42`.
 Couchbase allows you to increment and decrement these values atomically using a special [.api]`counter` operation.
-The example below (in Python) shows how to use counters:
+The example below (in Java) shows how to use counters:
 
-[source,python]
+[source,java]
 ----
->>> cb.counter('counter_id', delta=20, initial=100).value
-100L
->>> cb.counter('counter_id', delta=1).value
-101L
->>> cb.counter('counter_id', delta=-50).value
-51L
+bucket.counter("counter_id", 20, 100).content(); //100L
+bucket.counter("counter_id", 1).content(); //101L
+bucket.counter("counter_id", -51).content(); //51L
```

Comment:
I'd expect 101-51 to be //50L though, not // 51L.

Reply:
sorry. my mistake. I was supposed to do `bucket.counter("counter_id", -50).content(); //51L`. Will correct this.